sdk/internal/perf/perf.go

```diff
@@ -12,6 +12,25 @@ import (
 	"runtime"
 )
 
+func init() {
+	// Start with adding all of our arguments
+	flag.IntVar(&duration, "d", 10, "Duration of test in seconds.")
+	flag.IntVar(&duration, "duration", 10, "Duration of test in seconds.")
+
+	flag.StringVar(&testProxyURLs, "test-proxies", "", "test proxy URLs to target. This can be a semi-colon separated list for multiple proxies.")
```

Comment:
Tell me about your ambitions to have multiple text proxies? Is this failover or something interesting?

Reply:
(also 'comma separated' might be more intuitive, but not a big deal either way)

Reply:
semi-colon separated is the language agnostic way we've determined. This helps remove the test proxy as a bottleneck. If we have 8 parallel instances, we can also have 8 different proxies running and target one proxy per process.